apps/browser/src/auth/popup/sso.component.ts

```diff
@@ -70,6 +70,10 @@ export class SsoComponent extends BaseSsoComponent {
       this.win.close();
     };
 
+    super.onSuccessfulLoginTde = async () => {
+      await syncService.fullSync(true);
+    };
```

Comment:
Definitely a nit, but would be nice to have some consistency on these
```suggestion
    super.onSuccessfulLoginTde = () => {
      return syncService.fullSync(true);
    };
```